scripts/dockerfiles/Dockerfile.static

```diff
@@ -1,5 +1,7 @@
 ARG GO_VERSION
 FROM golang:$GO_VERSION as builder
+RUN  addgroup agentgroup
+RUN useradd -u 10001 agentuser -g agentgroup
```

Comment:
try `addgroup -u 1000 agentgroup` and `useradd -u 1000 agentuser -g agentgroup`

Reply:
added as suggested